build.gradle.kts

```diff
@@ -139,6 +141,9 @@ allprojects {
     // `options.release = X` makes sure that regardless of Java version used to run the compiler,
     // only Java X-compatible APIs are available to the compiled code.
     options.release.set(Integer.parseInt(targetJavaVersion.majorVersion))
+
+    // Add files from config/checker directory as inputs to java compilation (so that changes trigger recompilation)
```

Comment:
hmmm, but what about kotlin compile? is it enough to add it here only?

Reply:
It seems to me that the Checker Framework is for Java only? Also connected issue #1300 suggests that only JavaCompile should be affected. To be sure about this, @PawelLipski please refer.

Reply:
:yes:, Checker is for Java exclusively (and hence I'm reluctant to increase the presence of Kotlin in this project, given our reliance on Checker)

Reply:
Pls mention that in the code comment actually for clarity ☝🏻